SpatialGDK/Extras/schema/rpc_components.schema

```diff
@@ -18,16 +18,17 @@ type UnrealPackedRPCPayload {
 
 component UnrealClientRPCEndpoint {
```

Comment:
Sorry 😬, can you put a comment  next to "ready" or on top of the component stating what ready is for?

Reply:
Added comment.